autograder/cli/common.py

```diff
@@ -1,5 +1,13 @@
-HEADERS = ['email', 'name', 'role', 'lms-id']
-SYNC_HEADERS = HEADERS + ['operation']
+USER_HEADERS = ['email', 'name']
```

Comment:
Can you lexicographically re-order the constants , as that format is consistent throughout the repo